src/maps/language.tsx

```diff
@@ -9,13 +9,13 @@ type LanguageSyntaxHighlighterNames = {
 };
 
 const languageLabels: LanguageLabels = {
-  javascript: 'JS',
+  javascript: 'JavaScript',
```

Comment:
Does this also affect the local language buttons? I think we need a longLabel and a shortLabel for language and to select the one we need based on our use case

Reply:
yeah, you're right. fixed! 🙌